### ISSUE TITLE
MR for windows aria-live support

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#671]
This is from https://github.com/phetsims/scenery-phet/issues/490, and will need to patch as many sims as needed with 

https://github.com/phetsims/utterance-queue/commit/1f1e3c1e58c0b0ffbf7e64dc5820481bc570a372

@jessegreenberg does that sound right to you?

ration and proportion 1.0 has already been patched.

**Comment by jessegreenberg** [comment]
Yes, that sounds right to me. There is currently a MR in progress for https://github.com/phetsims/QA/issues/608, when that is finished we can begin this one.

**Comment by pixelzoom** [comment]
@zepumph @jessegreenberg can you fill me in on why this is relevant for scenery-phet?  The commit that you reference above concernes AriaHerald, and I see no occurrences of AriaHerald in scenery-phet code.

**Comment by jessegreenberg** [comment]
I am guessing this was opened in scenery-phet because the parent issue #490 is in scenery-phet. AriaHerald used to live in scenery-phet which is why #490 was opened here in 2019. This MR involves code now in utterance-queue, I will move the issue there.
